spec/factories/cards.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :card, class: Card do
+    original_text 'дом'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.